lib/content_security_policy/default.rb

```diff
@@ -8,6 +8,7 @@ class Default
     def initialize(base_url:)
       @base_url = base_url
       @directives = {}.tap do |directives|
+        directives[:upgrade_insecure_requests] = [] if SiteSetting.force_https
```

Comment:
Why is it an array? Shouldn't it be `1` instead? 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Upgrade-Insecure-Requests

Reply:
That's the header syntax. In CSP, `upgrade-insecure-requests;` is sufficient and an empty array will be converted to that in our parser. 

See also: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/upgrade-insecure-requests